controllers/admin/AdminBlockListingController.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+* 2007-2019 PrestaShop
+*
+* NOTICE OF LICENSE
+*
+* This source file is subject to the Open Software License (OSL 3.0)
+* that is bundled with this package in the file LICENSE.txt.
+* It is also available through the world-wide-web at this URL:
+* http://opensource.org/licenses/osl-3.0.php
+* If you did not receive a copy of the license and are unable to
+* obtain it through the world-wide-web, please send an email
+* to license@prestashop.com so we can send you a copy immediately.
+*
+* DISCLAIMER
+*
+* Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+* versions in the future. If you wish to customize PrestaShop for your
+* needs please refer to http://www.prestashop.com for more information.
+*
+* @author PrestaShop SA <contact@prestashop.com>
+* @copyright 2007-2019 PrestaShop SA
+* @license http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
+* International Registered Trademark & Property of PrestaShop SA
+**/
+
+class AdminBlockListingController extends ModuleAdminController
+{
+    /**
+     * __construct
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * ajaxProcessChangeBlockStatus
+     *
+     * @return bool
+     */
+    public function ajaxProcessChangeBlockStatus()
+    {
+        $iPsreassuranceId = (int) Tools::getValue('id');
+        $iStatus = (int) Tools::getValue('status');
+
+        $aUpdatedDatas = array(
+            'status' => $iStatus
```

Comment:
You can skip line 47 and directly put it here to save a line
```suggestion
            'status' => (int) Tools::getValue('status')
```